app/src/Controller/ArticlesController.php

```diff
@@ -60,6 +60,8 @@ public function add()
             }
             $this->Flash->error(__('The article could not be saved. Please, try again.'));
         }
+        $tags = $this->Articles->Tags->find('list')->all();
+        $this->set('tags', $tags);
         $this->set(compact('article'));
```

Comment:
We can simplify by using set & compact within 1 line only. 

Eg:

```
        $this->set(compact('article', 'tags'));
```

Please kindly refer https://www.php.net/manual/en/function.compact.php

Reply:
This was done in a future PR, this is the commit: https://github.com/combimauri/tutorial-cakephp/commit/d5dd52201ad9be51ccd66eae878ce96cdc5b2ead